govcd/api_vcd_test.go

```diff
@@ -58,6 +58,7 @@ const (
 	TestCreateExternalNetwork     = "TestCreateExternalNetwork"
 	TestDeleteExternalNetwork     = "TestDeleteExternalNetwork"
 	Test_LBServiceMonitor         = "Test_LBServiceMonitor"
+	Test_LBServerPool             = "Test_LBServerPool"
```

Comment:
In the sake of consistency, please remove the underscore from both variables and values.

Reply:
Done.